opennebula/shared_schemas.go

```diff
@@ -929,11 +944,17 @@ func flattenTemplate(d *schema.ResourceData, inheritedVectors map[string]interfa
 
 	// Set graphics to resource
 	if port != "" {
+		// passwd and random_passwd can't coexist
+		if passwd != "" && random_passwd != "" {
+			return fmt.Errorf("The arguments 'passwd' and 'random_passwd' can't coexist")
```

Comment:
You probably would prefer to use the native SDK option `ConflictWith`.

[Example](https://github.com/OpenNebula/terraform-provider-opennebula/blob/6c416689b850d4738246291902cc0e164defad23/opennebula/resource_opennebula_image.go#L105)

Reply:
had issues before so I switched the approach, trying this way one more time

Reply:
Many thanks. Do not hesitate to ask for help if you need some.